docs/markdown/Subprojects.md

```diff
@@ -77,3 +77,27 @@ subproject `b` and have `b` also use `a`.
 Meson ships with a dependency system to automatically obtain
 dependency subprojects. It is documented in the [Wrap dependency
 system manual](Wrap-dependency-system-manual.md).
+
+# Why all subproject should be inside a directory?
```

Comment:
I think the question being answered here is "Why must all subprojects be inside a single directory?"

Reply:
You are right, thank you.